examples/control_flow.rs

```diff
@@ -3,11 +3,9 @@
 use std::{thread, time};
 
 use simple_logger::SimpleLogger;
-use winit::{
-    event::{Event, KeyboardInput, WindowEvent},
-    event_loop::EventLoop,
-    window::WindowBuilder,
-};
+use winit::event::{Event, KeyboardInput, WindowEvent};
+use winit::event_loop::EventLoop;
+use winit::window::WindowBuilder;
```

Comment:
I personally quite like this grouping, are we missing an option?

Reply:
I dislike it, I find that it makes merge conflicts harder

Reply:
I've personally switched to `imports_granularity = "Item"` to make my life easier on that front.

Reply:
I personally hate the grouping current winit is using, and personally I don't have it in Wayland backend, since rustfmt doesn't enforce it. It just nearly impossible to read it and rebase. Or maybe I've just used to `Module` option.